osu.Game/Skinning/LegacyBeatmapSkin.cs

```diff
@@ -9,6 +9,9 @@ namespace osu.Game.Skinning
 {
     public class LegacyBeatmapSkin : LegacySkin
     {
+        // Null should be returned in the case of no colours provided to fallback into current skin's colours.
```

Comment:
this comment is no sense. null what? no colours provided to fallback?

Reply:
Was afraid of that (my explanation is shit). null should be returned if the beatmap skin doesn't provide any combo colours when trying to retrieve with `GetConfig(...)`, this will make `SkinProvidingContainer` retrieve from the current skin's (`SkinProvidingContainer.fallbackSource`) combo colours.